### DISCUSSION TITLE
always show spotify logo

### DISCUSSION THREAD

**Discussion by CharlesMougenot** [skuethe/MMM-Spotify#140] [General]
MMM-Spotify is a great module.
However I liked it better without spotify logo: if spotify is unused the logo uses space on the mirror for nothing. Maybe an option to hidden it would be nice.

**Comment by zynapsid** [comment]
It's mentioned in the readme that the module can send a notification to be hidden by another module when not in use, I've been trying to do this without success using MMM-Remote-Control, hopefully the author can shed some light on what I'm doing wrong, my config (unless this is one for the author of the other module, I'm pretty new to this so it's not clear):
`config: {
			notificationsOnResume:  [
									{
									notification: "SHOW",
									payload: "MMM-Spotify&apiKey=MYAPI",
									},
									],
			notificationsOnSuspend: [
									{
									notification: "HIDE",
									payload: "MMM-Spotify&apiKey=MYAPI",
									}
									],
			}`

**Comment by zynapsid** [comment]
OK to update my previous answer I found a comment by the author that the large Spotify logo is simply an image stored in the module folder: MMM-Spotify\Resources\spotify-xxl.png. I simply opened this in an image editor (paint.net on my PC) a deleted the logo part of the image so it was transparent and saved it and reuploaded it over the original version.  The profile switcher icon is still visible but I  think this can be disabled via the module config.

**Reply by CharlesMougenot** [reply]
Thank you for this solution, it works perfectly :-)

**Comment by skuethe** [comment]
Hey there, sorry for the late reply, I wasn't tracking new discussions in this repo.
I see that you already found a workaround for your question. I will add this as an possible improvement to future versions.
